src/ceaserCipher.js

```diff
@@ -0,0 +1,18 @@
+const alphabet = 'abcdefghijklmnopqrstuvwxyz';
+const cipher = (string, number) => {
+  const indexes = [];
+  string.split('').forEach((item) => {
```

Comment:
- [ ] [OPTIONAL] Splitting the string is not necessary, you can use the for of loop: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of